Signal/src/AppDelegate.m

```diff
@@ -268,30 +268,64 @@ - (BOOL)application:(UIApplication *)application
         NSString *filename = url.lastPathComponent;
         if ([filename stringByDeletingPathExtension].length < 1) {
             DDLogError(@"Application opened with URL invalid filename: %@", url);
+            [self showErrorAlertWithTitle:
+                      NSLocalizedString(@"EXPORT_WITH_SIGNAL_ERROR_TITLE",
+                          @"Title for the alert indicating the 'export with signal' attachment had an error.")
+                                  message:NSLocalizedString(@"EXPORT_WITH_SIGNAL_ERROR_MESSAGE_INVALID_FILENAME",
+                                              @"Message for the alert indicating the 'export with signal' file had an "
+                                              @"invalid filename.")];
             return NO;
         }
         NSString *fileExtension = [filename pathExtension];
         if (fileExtension.length < 1) {
             DDLogError(@"Application opened with URL missing file extension: %@", url);
+            [self showErrorAlertWithTitle:
```

Comment:
Having an error is better than failing silently, but we actually want to support this case right?